Sources/client-sdk-swift/config/TopicConfigurationProtocol.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+protocol TopicClientConfigurationProtocol {
+    var loggerFactory: MomentoLoggerFactoryProtocol { get }
+    var transportStrategy: TransportStrategyProtocol { get }
+    
+    func withTransportStrategy(transportStrategy: TransportStrategyProtocol) -> TopicClientConfigurationProtocol
+    func withClientTimeout(timeoutMillis: TimeInterval) -> TopicClientConfigurationProtocol
```

Comment:
`TimeInterval` is always specified in seconds, so this parameter shouldn't have `Millis` in it.

Reply:
Good catch